plugins/experimental/uri_signing/unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+add_executable(
+  uri_signing_test
+  uri_signing_test.cc
+  ${PROJECT_SOURCE_DIR}/jwt.cc
+  ${PROJECT_SOURCE_DIR}/common.cc
+  ${PROJECT_SOURCE_DIR}/config.cc
+  ${PROJECT_SOURCE_DIR}/cookie.cc
+  ${PROJECT_SOURCE_DIR}/jwt.cc
+  ${PROJECT_SOURCE_DIR}/match.cc
+  ${PROJECT_SOURCE_DIR}/normalize.cc
+  ${PROJECT_SOURCE_DIR}/parse.cc
+  ${PROJECT_SOURCE_DIR}/timing.cc
+  ${PROJECT_SOURCE_DIR}/uri_signing.cc
+)
+target_compile_definitions(uri_signing_test PRIVATE UNITTEST)
+target_include_directories(uri_signing_test PRIVATE ${PROJECT_SOURCE_DIR})
+target_link_libraries(
+  test_content_range
+  PRIVATE
+  catch2::catch2
+  jansson::jansson
+  cjose::cjose
+  PCRE::PCRE
```

Comment:
Here's PCRE again. It shouldn't be here. 🙂

Reply:
PCRE is linked in the autotools Makefile.inc.  Could it be an indirect depedency?

Reply:
Yes, but we should only need to list direct dependencies. CMake will link indirect dependencies for us, if it knows what they are. More concretely, if `PCRE` is listed as a `PUBLIC` dependency of a direct dependency, then it will add it to the link line.